CONTRIBUTING.md

```diff
@@ -34,15 +34,11 @@ If you don't have npm, it comes with node. To get it via homebrew:
 `brew install node`
 
 To add or update a dependency:
-- `go get -u` to update the dependencies or `go get {package}` to add a dependency
+- `cat GLOCKFILE | grep -oE '[^[:space:]]+\.[^[:space:]]+' | sort | uniq | xargs go get -u` to update the dependencies or `go get {package}` to add a dependency
```

Comment:
Ugh. We shouldn't encourage a complex shell pipeline like this. In general I think we should update one dependency at a time with `go get -u`; if you want to update everything at once you can `cd $GOPATH/src && go get -u ./...` (right?)`. I would also endorse adding a `glock update` command to do what you've described here.